UPGRADING.rst

```diff
@@ -10,6 +10,11 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#3497 Fix: API Gateway log is missing important fields
+======================================================
+
+Update personal deployments by running the ``make deploy`` command.
```

Comment:
This is implied. There is nothing special do to for upgrading a deployment so we won't need this section.